git-extra/git-update-git-for-windows

```diff
@@ -346,7 +346,7 @@ update_git_for_windows () {
 		echo "Downloading $filename" >&2
 	fi
 	curl -# -L -o $installer $download || return
-	start "" "$installer" //SILENT //NORESTART
+	start "" "$installer" //VERYSILENT //NORESTART
```

Comment:
As pointed out by @rimrul, this contradicts https://github.com/git-for-windows/build-extra/commit/9d3660bcb68c8dcbbc0a5e7db556ac1442f83f2a.

When essentially reverting a previous commit, it is _particularly_ important to make a good, convincing case that it should be reverted. This is typically done in the commit message. For guidance how to improve the commit message substantially, see https://github.blog/2022-06-30-write-better-commits-build-better-projects/.

Also, instead of leaving the comment about the [DCO](https://git-scm.com/docs/SubmittingPatches#dco) in the PR template, it would have made sense to heed the advice (i.e. to sign off on the commit) and delete the advice...

Reply:
@MPGaby23 are you prepared to make the changes we requested?